src/libstore/include/nix/store/s3.hh

```diff
@@ -1,7 +1,7 @@
 #pragma once
 ///@file
 #include "store-config-private.hh"
-#if NIX_WITH_S3_SUPPORT
+#ifdef NIX_WITH_S3_SUPPORT
```

Comment:
This seems inconsistent with other uses of this variable, which are all `#if ...`. It's defined as 
```
configdata_pub.set('NIX_WITH_S3_SUPPORT', aws_s3.found().to_int())
```
so I think `#if` should be correct.